### PR TITLE
Fixes #21868 - Refactored "troubleshooting" to docs link

### PR DESCRIPTION
**Pull request by ShimShtein**
For both VNC and SPICE console templates.

### CODE REVIEW

**Review by timogoebel — APPROVED** [review]

File: app/views/hosts/console/spice.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <% title "#{@console[:name]}" %>
 <%= title_actions(
         button_group(link_to(_("Ctrl-Alt-Del"), "#", :id => "sendCtrlAltDelButton", :onclick => 'sendCtrlAltDel()', :class => "btn btn-default"),
-                     link_to(_("Troubleshooting"), "http://theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#7.1NoVNC",
```
Comment (iNecas) [inline app/views/hosts/console/spice.html.erb:5]: the issue also suggests to rename "troubleshooting" to"documentation", which would be consistent with other places

**Review by tbrisker — COMMENTED** [review]
shouldn't this use `documentation_button` helper to be consistent with other documentation links?

**Comment by ShimShtein** [comment]
@tbrisker, it's either keeping all three items in a single button group, or ungrouping them. 
Here is an example on ungrouped buttons:
image
Personally, I think ungrouping is better, but I don't have strong opinion to either option.

**Comment by ShimShtein** [comment]
@Rohoover, Any thoughts? Originally it was a single buttongroup:

image

**Comment by Rohoover** [comment]
Ungroup, they're not related.

**Comment by ShimShtein** [comment]
Ungrouped for both VNC and spice.

File: app/views/hosts/console/spice.html.erb [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
                        link_to(_("New window"), "#", :id => "disconnect", :class => "btn btn-default", :onclick => 'connectXPI()', :title => _("Open Spice in a new window"))
                      end,
                      if @host
                        link_to_if_authorized(_("Back to host"), hash_for_host_path(:id => @host), :title => _("Back to host"), :class => 'btn btn-default')
                      end
-        )) %>
-
+                   ),
+                   documentation_button("7.1NoVNC",
+                     {:id => "troubleshootingButton", :class => "btn btn-default"})
```
Comment (tbrisker) [inline app/views/hosts/console/spice.html.erb]: Nitpick - I think the indentation here is a bit off, the documentation_button should be on the same level as the button group iiuc

**Review by tbrisker — CHANGES_REQUESTED** [review]

**Comment by ShimShtein** [comment]
Fixed spacing

**Comment by ShimShtein** [comment]
@tbrisker Wanna give it a quick run?

**Review by tbrisker — APPROVED** [review]
LGTM, Thanks @ShimShtein !

**Pull request merged**